src/main/java/core/mvc/resolver/PathVariableArgumentResolver.java

```diff
@@ -0,0 +1,51 @@
+package core.mvc.resolver;
+
+import core.annotation.web.PathVariable;
+import core.annotation.web.RequestMapping;
+import core.mvc.tobe.MethodParameter;
+import org.springframework.http.server.PathContainer;
+import org.springframework.web.util.pattern.PathPattern;
+import org.springframework.web.util.pattern.PathPatternParser;
+
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Map;
+
+public class PathVariableArgumentResolver implements MethodArgumentResolver {
+    private static final PathPatternParser PATH_PATTERN_PARSER = new PathPatternParser();
+
+    @Override
+    public boolean supportsParameter(MethodParameter parameter) {
+        return parameter.hasAnnotationInParameter(PathVariable.class);
+    }
+
+    @Override
+    public Object resolveArgument(MethodParameter parameter, HttpServletRequest request, HttpServletResponse response) {
+        RequestMapping rm = parameter.getMethod().getAnnotation(RequestMapping.class);
+        PathPattern.PathMatchInfo pathMatchInfo = getPathMatchInfo(rm, request.getRequestURI());
+
+//        if (Objects.isNull(pathMatchInfo)) {
+//            return getValueWithMatchingType(methodParameter.getParameterType(), null);
+//        }
+
+        Map<String, String> uriVariables = pathMatchInfo.getUriVariables();
+        String pathValue = parameter.getParameter().getAnnotation(PathVariable.class).value();
```

Comment:
디미터 법칙 위반
값을 get하지 말고 parameter에 메시지를 보내면 어떨까?

```suggestion
        String pathValue = parameter.getAnnotaionValue(PathVariable.class);
```